README.md

```diff
@@ -3,17 +3,80 @@
 [![CI](https://github.com/camunda/camunda-bpmn-js/workflows/CI/badge.svg)](https://github.com/camunda/camunda-bpmn-js/actions?query=workflow%3ACI)
 
 
-Embeddable Camunda modeling distributions based on [bpmn-js](https://github.com/bpmn-io/bpmn-js).
-
-🚧👷‍♀️👷‍♂️🚧
+Embeddable Camunda modeling distributions based on [bpmn-js](https://github.com/bpmn-io/bpmn-js). 
 
 ## Usage
 
-...
+This project is designed to deliver different BPMN modeling distributions. Instead of creating custom Modeler implementations based on [bpmn-js](https://github.com/bpmn-io/bpmn-js), choose one of the existing packages to mirror the modeling experience of [Camunda's modeling](https://github.com/camunda/camunda-modeler/) products.
+
+Use a pre-packaged version
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/camunda-bpmn-js@0.1.0/dist/assets/camunda-platform-modeler.css" />
+
+<script src="https://unpkg.com/camunda-bpmn-js@0.1.0/dist/camunda-platform-modeler.development.js"></script>
+```
+
+or include it via npm
+
+```js
+import BpmnModeler from 'camunda-bpmn-js/lib/camunda-platform/Modeler';
+
+import 'camunda-bpmn-js/dist/assets/camunda-platform-modeler.css';
+```
+
+into your node-style web-application.
```

Comment:
Why is it _node-style_?

Reply:
Like in a basic node application via `import`.

Copied it from the [`bpmn-js` introduction](https://github.com/bpmn-io/bpmn-js#installation). Maybe we could restructure it to make it clear "node-style" only refers to the second possibility.

Reply:
If I understand it correctly, by _node-style web-application_ we meant applications that are bundled. Is that correct? 

Anyway, I would say that you can _use pre-packaged distributions or install it via npm_. Most app starters like create-react-app already include webpack or rollup setup, so it should work even if don't know anything about bundling.

Reply:
or we can leave it as it is

Reply:
> use pre-packaged distributions or install it via npm

Let's go for that. Thanks :+1: